Cargo.toml

```diff
@@ -96,3 +96,5 @@ rpath = true
 [patch.crates-io]
 prost = { git = "https://github.com/mobilecoinofficial/prost", rev = "4e1905329369ca7a1cac3eda978ee9379167ee95" }
 prost-derive = { git = "https://github.com/mobilecoinofficial/prost", rev = "4e1905329369ca7a1cac3eda978ee9379167ee95" }
+dialoguer = { git = "https://github.com/mitsuhiko/dialoguer" }
+console = { git = "https://github.com/mitsuhiko/console" }
```

Comment:
This should tag the specific rev that you're looking for, in case they introduce other changes before they release 0.6.